opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -1217,6 +1295,72 @@ namespace Opm
         }
     }
 
+    template<typename TypeTag>
+    void
+    MultisegmentWell<TypeTag>::
+    updateIPRImplicit(const Simulator& ebos_simulator, WellState& well_state, DeferredLogger& deferred_logger)
+    {
+        // Compute IPR based on *converged* well-equation:
+        // For a component rate r the derivative dr/dbhp is obtained by 
+        // dr/dbhp = - (partial r/partial x) * inv(partial Eq/partial x) * (partial Eq/partial control_value)
+        // where Eq(x)=0 is the well equation setup with bhp control and primary varables x 
```

Comment:
This function is very similar to the one for StandardWells, and some of the same comments apply. Harmonize where appropriate.

Reply:
Done